test/utils/SafeTransferLib.t.sol

```diff
@@ -224,7 +224,8 @@ contract SafeTransferLibTest is Test {
         uint256 amount,
         bytes calldata brutalizeWith
     ) public brutalizeMemory(brutalizeWith) {
-        if (uint256(uint160(nonContract)) <= 18 || nonContract.code.length > 0) return;
+        vm.assume(uint256(uint160(nonContract)) > 18 && nonContract.code.length == 0);
```

Comment:
Note that this works too:
```suggestion
        vm.assume(nonContract > address(18) && nonContract.code.length == 0);
```

Reply:
Also, aren't pre-compiles between 0x1 and 0x9 ?

Reply:
That is correct. I wonder if the original code is attempting to account for future additions to precompiles?

Reply:
Addressed both comments